Configurations/10-main.conf

```diff
@@ -1269,8 +1269,17 @@ my %targets = (
         CC               => "cl",
         CPP              => '$(CC) /EP /C',
         CFLAGS           => "/W3 /wd4090 /nologo",
-        LDFLAGS          => add("/debug"),
         coutflag         => "/Fo",
+        LD               => "link",
+        LDFLAGS          => "/nologo /debug",
+        ldoutflag        => "/out:",
+        ldpostoutflag    => "",
+        ld_resp_delim    => "\n",
+        ld_wildcard_args => "setargv.obj",
```

Comment:
Is linking order important, or can this be at the very end?  If the latter, you could as well have this:

``` perl
        bin_ex_libs           => [ "setargv.obj" ],
```

and do the same in 50-cppbuilder.conf

One attribute less to care about...

Reply:
This would have worked with MSVC's `link.exe`, but unfortunately the [syntax of C++Builder's `ilink32.exe`](http://docwiki.embarcadero.com/RADStudio/Sydney/en/Using_ILINK32_and_ILINK64_on_the_Command_Line) expects all object files listed first, then the output `.exe`/`.dll` file, then the optional mapfile, and finally all the library files -- with commas between the groups. So `wildargs.obj` can't be placed at the end of the response file.

Reply:
Argh...

Might I then suggest a more generic option name?  `ex_objs`, perhaps (also permitting specializations such as `bin_ex_libs`, which is entirely what I expect to see in this PR, then)?

Reply:
Managed to `combine()` it in `bin_lflags`.

Reply:
That's an interesting hack, but yeahok, that works